airbyte-integrations/connectors/source-zendesk-talk/source_zendesk_talk/source.py

```diff
@@ -1,12 +1,94 @@
 #
 # Copyright (c) 2021 Airbyte, Inc., all rights reserved.
 #
+from datetime import datetime
+from typing import Any, List, Mapping, Tuple
 
+from airbyte_cdk import AirbyteLogger
+from airbyte_cdk.models import ConnectorSpecification, DestinationSyncMode, SyncMode
+from airbyte_cdk.sources import AbstractSource
+from airbyte_cdk.sources.streams import Stream
+from pydantic import BaseModel, Field
+from requests.auth import HTTPBasicAuth
+from source_zendesk_talk.streams import (
+    AccountOverview,
+    Addresses,
+    AgentsActivity,
+    AgentsOverview,
+    CallLegs,
+    Calls,
+    CurrentQueueActivity,
+    GreetingCategories,
+    Greetings,
+    IVRMenus,
+    IVRRoutes,
+    IVRs,
+    PhoneNumbers,
+)
 
-from base_python import BaseSource
 
-from .client import Client
+class ConnectorConfig(BaseModel):
+    class Config:
+        title = "Zendesk Talk Spec"
 
+    subdomain: str = Field(
+        description="The subdomain for your Zendesk Talk.",
+    )
+    access_token: str = Field(
+        description='The value of the API token generated. See the <a href="https://docs.airbyte.io/integrations/sources/zendesk-talk">docs</a> for more information.',
+        airbyte_secret=True,
+    )
+    email: str = Field(description="The user email for your Zendesk account.")
+    start_date: datetime = Field(
+        description="The date from which you'd like to replicate data for Zendesk Talk API, in the format YYYY-MM-DDT00:00:00Z.",
+        pattern="^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}Z$",
```

Comment:
we can probably drop the `T00:00:00Z` convention. YYYY-MM-DD is probably enough. WDYT? Maybe just change the `pattern` to make that part optional

Reply:
nice idea, btw, do we still need a pattern here or it is legacy?

Reply:
`pattern` is still relevant because it allows validation here and in UI, just suggesting we make the pattern simpler